### ISSUE TITLE
[Accessibility] Slide Viewer: Tooltip inaccessible on hover

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#994]
**Describe the bug**
In the Annotation Assistant modal within the Preset Labels function of the Slide Viewer, the radius, threshold, and min overlap texts show a tooltip on hover. This tooltip can be dismissed and remains persistent, but it is not accessible via hover; it disappears upon hovering over it, making its content inaccessible to screen readers.

**To Reproduce**
1. Go to 'Slides'
2. Open a slide
3. Click on 'Present labels' icon
4. Go to 'Annotation assistant'
5. Hover over the texts "Radius", "Threshold", and "Min overlap". Notice that a tooltip appears.
6. Try to hover over the tooltip and notice the tooltip disappear on hover. This behavior makes the content of the tooltip inaccessible to screen readers.

**Expected behavior**
Content should not disappear on hover so that screen readers can access it.

**Screenshots**
![tooltip inaccessible on hover](https://github.com/camicroscope/caMicroscope/assets/122698422/f63a954d-4432-400b-98e3-0eb3e04ab1fc)

**Comment by rajku-dev** [comment]
i can fix this issue.
